test/functionalTest/cases/1156_qfilters_and_compounds/qfilters_and_compounds_deeper.test

```diff
@@ -653,7 +653,7 @@ Fiware-Correlator: REGEX([0-9a-f\-]{36})
                             "F": false, 
                             "N": null, 
                             "T": true, 
-                            "f": 3.14,
+                            "f": 3.14REGEX([01]*),
```

Comment:
This is the "rollback" of the .test that I wrongly changed in my previous PR, isn't it?